v2_1/hrmanifest.xsd

```diff
@@ -0,0 +1,432 @@
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="addressTypeCode">
+    <!-- HM = Home; OTH = Other; WRK = Work -->
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="3"/>
+      <xs:pattern value="HM|OTH|WRK"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="nameCode">
+    <!-- OTH = Other; PRFR = Preferred; PRM = Primary -->
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="4"/>
+      <xs:pattern value="OTH|PRFR|PRM"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="prefix">
+    <!-- Ms;Mrs;Mr;Dr -->
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="3"/>
+      <xs:pattern value="(Ms|Mrs|Mr|Dr)?"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="suffix">
+    <!-- Jr;Sr;Mr;Md -->
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="2"/>
+      <xs:pattern value="(Jr|Sr|Mr|Md)?"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="phoneType">
+    <!-- FAX = Facsimile; HM = Home; MBL = Mobile; OTH = Other; WRK = Work -->
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="3"/>
+      <xs:pattern value="FAX|HM|MBL|OTH|WRK"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="phoneNumber">
+    <!-- e.g.: 555-555-1212 format required by application validation rules -->
+    <xs:restriction base="xs:string">
+      <xs:pattern value="\d{3}-\d{3}-\d{4}"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="emailType">
+    <!-- HM = Home; OTH = Other; WRK = Work -->
+    <xs:restriction base="xs:string">
+      <xs:maxLength value="3"/>
+      <xs:pattern value="HM|OTH|WRK"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="emailAddress">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="([a-zA-Z0-9_\-\.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([a-zA-Z0-9\-]+\.)+))([a-zA-Z]{2,5}|[0-9]{1,3})(\]?)"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <xs:simpleType name="year">
+    <xs:restriction base="xs:int">
+      <xs:totalDigits value="4"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+  <!--
+  +=======================+====================================+
+  | CITIZENSHIP_TYPE_CODE | DESCRIPTION                        |
+  +=======================+====================================+
+  |                     1 | US CITIZEN OR NONCITIZEN NATIONAL  |
+  |                     2 | PERMANENT RESIDENT OF US           |
+  |                     3 | NON US CITIZEN WITH TEMPORARY VISA |
+  |                     4 | PERMANENT RESIDENT OF US PENDING   |
+  |                     5 | OTHER                              |
+  +=======================+====================================+ -->
+  <xs:simpleType name="citizenshipType">
+    <xs:restriction base="xs:int">
+      <xs:totalDigits value="1"/>
+      <xs:minInclusive value="1"/>
+      <xs:maxInclusive value="5"/>
```

Comment:
Hmm...how about just max being 16. That means the xsd might validate potentially invalid values, but those will all be caught when the xml is uploaded? I'm wondering if we want to relax this even more as each institution can add any citizenship type they want and seems like the xsd should at least allow those?